### ISSUE TITLE
rvc always shows 'Success' even when there are no active connections

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#42]
```sh
$ src/rvc status
[ { "name": "stagingfoobarbaz", "status": "DISCONNECTED", "timestamp": 1503303694, "network": { "in-total": 0, "out-total": 0 } } ]
$ src/rvc disconnect all
Success
```

Maybe show something like 'no active connections' (in JSON) when there are no active connections.

**Comment by erikbor** [comment]
Also the return code will always be '0':

```sh
$ rvc disconnect all
Success
$ echo $?
0
$ rvc disconnect all
Success
$ echo $?
0
```

**Comment by jjr840430** [comment]
@erikbor The commands, 'connect' and 'disconnect' will work by async mode.
I think the result of 'disconnect all' doesn't have any meanings on the real environment.

**Comment by ronaldtse** [comment]
I think `roc connect/disconnect` should be synchronous to be able to display the results? At least it could return the current connection results from `rvcd` showing the connections that are "shutting-down".

Otherwise the returned "status" would have no meaning...

**Comment by jjr840430** [comment]
@ronaldtse At practice, we may assume the following case:

If UI connects to a VPN, then the request should be returned soon, because connect will be take for a while.
If it waits until connection has completed, then UI will be hang until then.

It's effiecienty way that rvcd daemon will notice the connection result to UI after the connection has completed.

Otherwise, UI may repeat getting connection status until detect the change of it.

**Comment by ronaldtse** [comment]
@jjr840430 indeed, there are 3 components here:

1. Caller (person or UI)
2. Relay (`rvcd`)
3. Connection handler (`openvpn`)

By synchronous, what I meant was that `rvc connect / disconnect` should be synchronous to `rvcd` (1 and 2), meaning that `rvcd` should acknowledge the response from 1 that it will start shutting down the connections, then the `rvc` command returns.

That's why I mentioned that the changing connections should be returned, e.g.,
```json
{
  "connections": [
    "my-connection": {
      "status": "DISCONNECTING"
      ...
    }
  ]
}
```

**Comment by jjr840430** [comment]
@ronaldtse OK, I will modify as you mentioned. Thanks.

**Comment by jjr840430** [comment]
Fixed on #52
